scipy/stats/kde.py

```diff
@@ -170,14 +171,19 @@ def __init__(self, dataset, bw_method=None):
         self.d, self.n = self.dataset.shape
         self.set_bandwidth(bw_method=bw_method)
 
-    def evaluate(self, points):
-        """Evaluate the estimated pdf on a set of points.
+    def evaluate(self, points, lb=None, ub=None):
+        """Evaluate the estimated pdf on a set of points using a Gaussian
+        Kernel, boundary effects corrected using the reflection method.
 
         Parameters
         ----------
         points : (# of dimensions, # of points)-array
             Alternatively, a (# of dimensions,) vector can be passed in and
             treated as a single point.
+        lb : Lower boundary for the estimated pdf, integer for unidimensional
+        distributions, array_like for multivariate ones
```

Comment:
Yeap, "integer" is typo sorry about that and for the sloppy docstring format. I will fix it ASAP. Thanks for the comments.